source/install/self-managed-changelog.md

```diff
@@ -4,13 +4,143 @@
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Added ``EnableInlineLatex`` to add support for inline Latex rendering.
+ - Under ``JobSettings`` in ``config.json``
+   - Added ``CleanupJobsThresholdDays``. This defines the time gap in hours beyond which older jobs will be removed. Default is -1 which means the feature is disabled. Setting to 0 will clean all completed jobs.
+   
+#### Database Changes
+ - Extended the maximum size to 256 characters for the following database columns:
+    - ``Sessions.Roles``
+    - ``ChannelMembers.Roles``
+    - ``TeamMembers.Roles``
+
+### API Changes
+ - Added a new API endpoint ``POST /api/v4/posts/search`` to perform searches across all channels.
+ 
+### Go Version
+ - v6.1 is built with Go ``v1.16.7``.
+
+### Open Source Components
+ - Added ``fast-deep-equal``, ``luxon``, and ``react-window-infinite-loader`` to https://github.com/mattermost/mattermost-webapp.
+ - Added ``@mattermost/react-native-paste-input`` to https://github.com/mattermost/mattermost-mobile.
+
+### Known Issues
+ - Member type is missing from autocomplete [MM-38989](https://mattermost.atlassian.net/browse/MM-38989).
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CMD+/`` does not close shortcuts modal [MM-38971](https://mattermost.atlassian.net/browse/MM-38971).
+ - LDAP Sync job inserting invalid NULL unicode character into job's Data column [MM-38711](https://mattermost.atlassian.net/browse/MM-38711).
+ - ``Ctrl/Cmd+Shift+A`` shortcut does not open **Account Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
+ - Close button on invite people page is incorrectly themed [MM-37852](https://mattermost.atlassian.net/browse/MM-37852).
+ - Known issues related to the Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - Adding an at-mention at the start of a post draft and pressing the leftwards or rightwards arrow can clear the post draft and the undo history [MM-33823](https://mattermost.atlassian.net/browse/MM-33823).
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
+ - Searching stop words in quotation marks with Elasticsearch enabled returns more than just the searched terms.
+ - The team sidebar on the desktop app does not update when channels have been read on mobile.
+ - Slack import through the CLI fails if email notifications are enabled.
+ - Push notifications don't always clear on iOS when running Mattermost in High Availability mode.
+
+### Contributors
```

Comment:
@puerco The below list of contributors also includes a list of translators that Carrie generated from Weblate. Carrie was also able to identify the URLs to most of the translators's Weblate profiles', except a few of them which you can see in this preview https://github.com/mattermost/docs/blob/126e2d409e8cc78aee86e0e3105770a2a851eaf5/source/install/self-managed-changelog.md#contributors.

Do you have admin-level access in Weblate to find the missing profile links? Please DM me for any questions.

Reply:
I think I can complete the list by getting the GitHub logins from weblate. Do you want me to push a commit to your branch?

Reply:
@puerco I think that should be fine, thank you

Reply:
Ready @amyblais I've managed to complete all links except one:  Erik Pfeiffer